Server/htdocs/AppController/commands_RSM/api/v2/items/getItemFromProperty.php

```diff
@@ -0,0 +1,172 @@
+<?php
+//****************************************************************************************
+//Description:
+//    Retrieves an item of the specified itemType with the associated values
+//
+//  REQUEST BODY
+//  {
+//    itemType: itemType to retrieve (for example: the itemType of crm-accounts)
+//    filterProperty: property of another itemType related with the first one (for example: the property 'client' into invoices)
+//    filterPropertyID: itemID of the filter property (for example: The identifier of the invoice from which we get the client)
+//  }
+//****************************************************************************************
+
+require_once '../../../utilities/RStools.php';
+require_once '../../../utilities/RSMverifyBody.php';
+setAuthorizationTokenOnGlobals();
+checkCorrectRequestMethod('GET');
+
+require_once '../../../utilities/RSdatabase.php';
+require_once '../../../utilities/RSMitemsManagement.php';
+require_once '../../api_headers.php';
+
+// Definitions
+$requestBody = getRequestBody();
+verifyBodyContent($requestBody);
+
+$clientID = getClientID();
+$RStoken =  getRStoken();
+$RSuserID =  getRSuserID();
+
+$itemType = $requestBody->itemType;
+$filterProperty   = $requestBody->filterProperty;
+$filterPropertyID = $requestBody->filterPropertyID;
+
+//translateIDs
+$translateIDs = false;
+if (isset($requestBody->translateIDs) && $requestBody->includeCategories) {
+    $translateIDs = true;
+}
+
+$properties   = array();
+$attributes   = array();
+$results      = array();
+
+// get type of the filterProperty
+$propertyType = getPropertyType($filterProperty, $clientID);
+
+// if filterProperty is unsupported type return empty response
+if (!isSingleIdentifier($propertyType) && !isMultiIdentifier($propertyType)) {
+    returnJsonMessage(200, '');
```

Comment:
Add a debug mode clause to get more details in the log or in the answer, if debug mode is enabled